client/components/grade.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Grade(props) {
+  return (
+    props.grades.map((objectGrades, index) =>
+      <tr key={index}>
+        <td>{objectGrades.name}</td>
+        <td>{objectGrades.course}</td>
+        <td>{objectGrades.grade}</td>
+      </tr>
+    )
```

Comment:
The `.map()` should happen in your `GradeTable` component. The `Grade` component is only responsible for render one table row.